src/18F340.c

```diff
@@ -1299,7 +1299,7 @@ EvtScript EVS_PlayerDies = {
         EVT_CALL(SetActorYaw, ACTOR_PLAYER, LVar0)
         EVT_WAIT(1)
     EVT_END_LOOP
-    EVT_CALL(EnablePlayerBlur, 0)
+    EVT_CALL(EnablePlayerBlur, IDLE_SCRIPT_ENABLE)
```

Comment:
This should be a separate enum specific to this function: BLUR_DISABLE|BLUR_ENABLE|BLUR_RESET

Reply:
should `IDLE_SCRIPT_RESTART` become `IDLE_SCRIPT_RESET`?

Reply:
Its not necessary for the enums to be consistent, as they're not really related. The handler for `IDLE_SCRIPT_RESTART` calls `restart_script`, better to be consistent with that.